MAUI/Effects-View/Effects/Rotation.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```

Comment:
The .NET MAUI Rotate Animation Effects | Effects View control | Syncfusion

Reply:
Updated.